sdk/translation/azure-ai-translation-text/samples/sample_text_translation_translate.py

```diff
@@ -0,0 +1,307 @@
+        translation = response[0] if response else None
+
+        if translation:
+            for translated_text in translation.translations:
+                print(f"Text was translated to: '{translated_text.to}' and the result is: '{translated_text.text}'.")
+
+    except HttpResponseError as exception:
+        print(f"Error Code: {exception.error.code}")
+        print(f"Message: {exception.error.message}")
+    # [END get_text_translation_entity]
+
+def get_text_translation_profanity():
+    # [START get_text_translation_profanity]
+    try:
+        profanity_action = ProfanityAction.MARKED
+        profanity_maker = ProfanityMarker.ASTERISK
+        target_languages = ["cs"]
+        input_text_elements = [ InputTextItem(text = "This is ***.") ]
+
+        response = text_translator.translate(content = input_text_elements, to = target_languages, profanity_action=profanity_action, profanity_marker=profanity_maker)
+        translation = response[0] if response else None
+
+        if translation:
+            detected_language = translation.detected_language
+            if detected_language:
+                print(f"Detected languages of the input text: {detected_language.language} with score: {detected_language.score}.")
+            for translated_text in translation.translations:
+                print(f"Text was translated to: '{translated_text.to}' and the result is: '{translated_text.text}'.")
+
+    except HttpResponseError as exception:
+        print(f"Error Code: {exception.error.code}")
+        print(f"Message: {exception.error.message}")
+    # [END get_text_translation_profanity]
+
+def get_text_translation_alignment():
+    # [START get_text_translation_alignment]
+    try:
+        include_alignment = True
+        target_languages = ["cs"]
+        input_text_elements = [ InputTextItem(text = "The answer lies in machine translation.") ]
+
+        response = text_translator.translate(content = input_text_elements, to = target_languages, include_alignment = include_alignment)
+        translation = response[0] if response else None
+
+        if translation:
+            detected_language = translation.detected_language
+            if detected_language:
+                print(f"Detected languages of the input text: {detected_language.language} with score: {detected_language.score}.")
+            for translated_text in translation.translations:
+                print(f"Text was translated to: '{translated_text.to}' and the result is: '{translated_text.text}'.")
+                if (translated_text.alignment):
+                    print(f"Alignments: {translated_text.alignment.proj}")
+
+    except HttpResponseError as exception:
+        print(f"Error Code: {exception.error.code}")
+        print(f"Message: {exception.error.message}")
+    # [END get_text_translation_alignment]
+
+def get_text_translation_sentence_length():
+    # [START get_text_translation_sentence_length]
```

Comment:
Does it make sense to separate these different snippets into separate files?

Reply:
It does, however I think for the scope of this PR we should leave it since there's 1:1 parity with the function definitions in the previous Sample2_Translate.md file.